identifier/vite.config.js

```diff
@@ -23,13 +48,18 @@ export default defineConfig((env) => {
     },
     plugins: [
       react(),
-      env.mode !== 'test' && checker({
-        typescript: true,
-        eslint: {
-          lintCommand: 'eslint --max-warnings=0 src',
-        },
+      legacy({
+        targets: ['edge 18'],
       }),
+      env.mode !== "test" &&
+        checker({
+          typescript: true,
+          eslint: {
+          lintCommand: 'eslint --max-warnings=0 src',
```

Comment:
This looks like there is an indentation problem - please check.

Reply:
Updated this with an identation.